Directory.Build.props

```diff
@@ -10,6 +10,43 @@
   -->
   <PropertyGroup>
     <ReactNativeWindowsDir>$(MSBuildThisFileDirectory)vnext\</ReactNativeWindowsDir>
+
+    <Platform Condition="'$(Platform)' == ''">x64</Platform>
+    <Configuration Condition="'$(Configuration)' == ''">Debug</Configuration>
+
+    <RootIntDir>$(ReactNativeWindowsDir)build</RootIntDir>
+    <RootOutDir>$(ReactNativeWindowsDir)target</RootOutDir>
+    <!-- Visual Studio forces using 'Win32' for the 'x86' platform. -->
+    <BaseIntDir Condition="'$(Platform)' == 'Win32'">$(RootIntDir)\x86\$(Configuration)</BaseIntDir>
+    <BaseIntDir Condition="'$(Platform)' != 'Win32'">$(RootIntDir)\$(Platform)\$(Configuration)</BaseIntDir>
+    <BaseOutDir Condition="'$(Platform)' == 'Win32'">$(RootOutDir)\x86\$(Configuration)</BaseOutDir>
+    <BaseOutDir Condition="'$(Platform)' != 'Win32'">$(RootOutDir)\$(Platform)\$(Configuration)</BaseOutDir>
   </PropertyGroup>
 
-</Project>
\ No newline at end of file
+  <PropertyGroup Label="NuGet">
+    <MSBuildProjectExtensionsPath Condition="'$(ProjectName)' != ''">$(RootIntDir)\ProjectExtensions\$(ProjectName)\</MSBuildProjectExtensionsPath>
+    <MSBuildProjectExtensionsPath Condition="'$(ProjectName)' == ''">$(RootIntDir)\ProjectExtensions\$(MSBuildProjectName)\</MSBuildProjectExtensionsPath>
```

Comment:
This is the path for the PackageReference intermediate files (absolute paths for the machine, etc).
I'm ensuring it's placed within the intermediate artifact directory, instead of next to the sources (same principle as defining the `build` intermediate output path).